aaad/utils.py

```diff
@@ -44,3 +44,12 @@ def getAllowedNamespacePatterns(act_as, permissions):
 
     return allowed_namespace_patterns
 
+def merge_dicts(dict1, dict2):
```

Comment:
There's a `.update()` method on all dictionaries, so you can do `existing_dict.update(dict_with_more_stuff)`.